### PR TITLE
Feat/tutorial video

### PR DESCRIPTION
**Pull request by itsalaidbacklife**
## Please check the following

- [ x ] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle#run-the-tests))
- [ x ] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle#linting-formatting))
- For New Features:
  - [ ] Have tests been added to cover any new features or fixes?
  - [ ] Has the documentation been updated accordingly?

## Please describe additional details for testing this change
Adds tutorial video to rules page and block quote testimonial to login
image

image

### CODE REVIEW

File: client/js/views/Rules.vue [hunk 1]
```diff
@@ -380,6 +398,24 @@ export default {
...
+          return 800;
+        case 'xl':
+          return 1000;
+        default:
+          return 560;
+      }
+    },
+    videoHeight() {
+      return (this.videoWidth * 315) / 560;
+    },
```
Comment (seriouslysean) [inline client/js/views/Rules.vue:418]: Would it be better to set a padding-top percent and make the video responsive in nature?

**Review by seriouslysean — APPROVED** [review]
One comment, rest looks fine to me 👍🏼

File: client/js/views/Rules.vue (hunk 1 above)
Comment (seriouslysean) [reply client/js/views/Rules.vue:418]: https://codepen.io/seriouslysean/pen/RwMwmYq

Comment (itsalaidbacklife) [reply client/js/views/Rules.vue]: absolutely, this is better. Done

**Pull request merged**